### PR TITLE
Add pwd

### PR DESCRIPTION
**Pull request by botatoes**
pwd.md

### CODE REVIEW

File: _commands/basics/pwd.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+---
+
+pwd
+--
+
+`pwd` stands for print working directory. `pwd` outputs the full pathname of the current work director.
```
Comment (ppannuto) [inline _commands/basics/pwd.md]: typo: I think this should read "current working directory"

File: _commands/basics/pwd.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+$ cd test && pwd
+/home/bo/Desktop/test
+$ /bin/pwd
+/home/bo/Desktop
+$ /bin/pwd -L
+/home/bo/Desktop/test
+~~~
+
+##### Break it down
+* `--logical` If the contents of the environment variable 'PWD' provide an absolute name of the current directory with no '.' or '..' components, but possibly with symbolic links, then output those contents. Otherwise, fall back to default -P handling.
```
Comment (ppannuto) [inline _commands/basics/pwd.md]: This reads very densely, like a man page. We're trying to be a little more accessible. Can you expand on this text a little to make it easier to understand?

File: _commands/basics/pwd.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+$ ln -s . test
+$ cd test && pwd
+/home/bo/Desktop/test
+$ /bin/pwd
+/home/bo/Desktop
+$ /bin/pwd -L
+/home/bo/Desktop/test
+~~~
+
+##### Break it down
```
Comment (ppannuto) [inline _commands/basics/pwd.md]: It's not clear from this where the "other" pwd is coming from. Could you add an explanation about that?

File: _commands/basics/pwd.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+* `--logical` If the contents of the environment variable 'PWD' provide an absolute name of the current directory with no '.' or '..' components, but possibly with symbolic links, then output those contents. Otherwise, fall back to default -P handling. 
+* The `-L` option displays the logical current working directory. Built in 'pwd' includes symlinks by default except when `-P` is used. `/bin/pwd` ignore symlinks and prints out the actual directory. For `/bin/pwd` to get the same result as the built in `pwd`, you must use the `-L` option. 
+
+#### `pwd -P`
+
+~~~ bash
+$cd ~/bin
```
Comment (ppannuto) [inline _commands/basics/pwd.md]: style nit: add a space between "$" and "cd"

**Comment by ppannuto** [comment]
Great start! I added some comments for above for a few things that could be improved. If you push new commits to the branch you created the pull request from, it will update automatically.

Could you also add a break so that the preview on the page only includes the top bit, currently it looks like this:
screen shot 2016-03-13 at 15 57 00

File: _commands/basics/pwd.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+
+<!--more-->
+
+### Useful Options / Information 
+
+#### `pwd` vs `/bin/pwd`
+* `/bin/pwd` ignores symlinks and prints actual directory by default. 
+* `pwd` is a shell builtin
+* `pwd` is a variable that is `/bin/pwd`
```
Comment (themoh) [inline _commands/basics/pwd.md]: This third bullet needs to be a bit more clear

File: _commands/basics/pwd.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+~~~ bash
+$ cd ~/bin
+$ pwd
+/home/bo/bin
+$ pwd -P
+/home/bo/realdata/scripts/utils
+~~~
+
+##### Break it down
+* `--physical` Print a fully resolved name for the current directory. That is, all components of the printed name will be actual directory names--none will be symbolic links.
```
Comment (themoh) [inline _commands/basics/pwd.md]: "--physical or -P ...", stick to your conventions

**Comment by ppannuto** [comment]
Looks like this is just waiting on the comments from @themoh and then this should be good to go.

**Comment by the-alex** [comment]
It looks like we got those changes referenced by themoh, but no one went to merge. I'll do that as soon as we get your name and uniquename.

**Pull request merged**